cmake/config-build.cmake

```diff
@@ -11,13 +13,34 @@ if(NOT RTS_BUILD_ZEROHOUR AND NOT RTS_BUILD_GENERALS)
     message("You must select one project to build, building Zero Hour by default.")
 endif()
 
+add_feature_info(CoreTools RTS_BUILD_CORE_TOOLS "Build Core Mod Tools")
+add_feature_info(CoreExtras RTS_BUILD_CORE_EXTRAS "Build Core Extra Tools/Tests")
 add_feature_info(ZeroHourStuff RTS_BUILD_ZEROHOUR "Build Zero Hour code")
 add_feature_info(GeneralsStuff RTS_BUILD_GENERALS "Build Generals code")
 add_feature_info(InternalBuild RTS_BUILD_OPTION_INTERNAL "Building as an \"Internal\" build")
 add_feature_info(ProfileBuild RTS_BUILD_OPTION_PROFILE "Building as a \"Profile\" build")
 add_feature_info(DebugBuild RTS_BUILD_OPTION_DEBUG "Building as a \"Debug\" build")
 add_feature_info(AddressSanitizer RTS_BUILD_OPTION_ASAN "Building with address sanitizer")
 
+if(RTS_BUILD_ZEROHOUR)
```

Comment:
Not sure if this is necessary, but this is what it effectively did before this change too.